BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/provider/DeviceLocationPredictorTest.java

```diff
@@ -0,0 +1,19 @@
+package com.nexenio.bleindoorpositioning.location.provider;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by leon on 31.01.18.
+ */
+public class DeviceLocationPredictorTest {
+
+    @Test
+    public void calculateBearing_validLocations_accurateBearing() throws Exception {
+        assertEquals((int) DeviceLocationPredictor.calculateBearing(43.682213,-70.450696, 43.682194, -70.450769),250);
```

Comment:
Use the already available locations from `LocationTest` if you test locations.

Bearing / Angle calculation is tested at `LocationTest.getAngleTo_validLocations_correctAngles`.